Alignment-INDOT/readme.md

```diff
@@ -0,0 +1,60 @@
+### Intent
+
+This UT establishes a road alignment for the other INDOT unit tests
+
+It is based on the storylines [INDOT IR_SLRH-C](https://app.box.com/file/731742565663?s=x8prri9nbc1n0x2bjaezeqxvkr9o03sw) storyline
+
+### Prerequisites
+
+None
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+
+- Spatial structure
+    - `IfcSite`
+- Alignment
+    - `IfcAlignment`
+    - `IfcAlignmentHorizontal`
+    - `IfcAlignmentVertical`
+    - `IfcHorizontalSegment`
+    - `IfcVerticalsegment`
+- Geometry
+    - `IfcCompositeCurve`
+    - `IfcCurveSegment`
+    - `IfcLine`
+    - `IfcCircle`
+    - `IfcGradientCurve`
+    - `IfcPolynomialCurve`
+
+### Situation
+
+The figure below illustrates the horizontal and vertical alignments.
+
+![](./Illustration.PNG)
```

Comment:
For the sake of automatic report production, please always use "one folder up, one folder down" for the figures:

```suggestion
![](../Alignment-INDOT/Illustration.PNG)
```